ui.go

```diff
@@ -401,7 +401,7 @@ func (win *win) printDir(screen tcell.Screen, dir *dir, selections map[string]in
 			}
 		}
 
-		if i == dir.pos {
+		if i == dir.pos && gOpts.hasfocus {
 			st = st.Reverse(true)
 		}
```

Comment:
Thank you for making this! I'm using it, and it's nice.

However, I would personally prefer something like the following here:

```
		if i == dir.pos {
			st = st.Reverse(true)
			if dim_cursor || !gOpts.hasfocus {
				st = st.Foreground(tcell.ColorDarkGray)
			}
		}
```

I think the grey color looks nicer than a complete absence of a selection. Of course, in the ideal world the color would be configurable; I'm not sure if this can be done through the `LS_COLORS` variable or somehow else.

Reply:
(To be clear, I have no official review powers here).

Reply:
I switched to `tcell.ColorGray`. It should support more terminal and light themes better.

Reply:
I implemented it this way to behave like a cursor in tmux - there is only one active cursor, no dimming.

So if you prefer dimming I'd like your PR to have options ;-)

Reply:
I see. Here I was thinking dimming was strictly superior. Do you think dim cursors in unfocused windows are distracting? Do you not like them in general?

I was hoping to avoid an explosion of options, not sure what the best way is. One possibility is to have an on/off `focusevents` option and a `dimcursor` option with possible values `gray`, `hide`,  and `same`  that would govern what the "dimmed" cursor looks like both when focus is lost and for the directory preview (https://github.com/gokcehan/lf/pull/924.).

Reply:
> Do you think dim cursors in unfocused windows are distracting? 
Yes, too much noise IMO.